cpp/tests/structure/coarsen_graph_test.cpp

```diff
@@ -238,6 +238,14 @@ class Tests_CoarsenGraph
   virtual void SetUp() {}
   virtual void TearDown() {}
 
+  template <typename vertex_t, typename edge_t, typename weight_t, bool store_transposed>
+  void run_current_test(
+    std::tuple<CoarsenGraph_Usecase const&, input_usecase_t const&> const& param)
+  {
+    run_current_test<vertex_t, edge_t, weight_t, store_transposed>(std::get<0>(param),
+                                                                   std::get<1>(param));
+  }
```

Comment:
As I mentioned above, should we keep both versions?

Reply:
Fixed in latest push.